packages/@aws-cdk/aws-codepipeline-actions/lib/cloudformation/pipeline-actions.ts

```diff
@@ -540,3 +540,14 @@ interface StatementTemplate {
 }
 
 type StatementCondition = { [op: string]: { [attribute: string]: string } };
+
+function parseCapabilities(capabilities: any): any {
```

Comment:
Can we avoid using `any` here? I would rather those be tightly typed as much as possible.

Reply:
Yes, sorry for missing out on this one. I've updated the function.